expat/test-driver-wrapper.sh

```diff
@@ -40,4 +38,4 @@ while [[ ${1} != '--' ]]; do
 done
 shift  # drop "--"
 
-exec "${top_srcdir}"/conftools/test-driver "${test_driver_args[@]}" "${top_builddir}"/run.sh "$@"
+exec "${EXPAT_SRCDIR}"/conftools/test-driver "${test_driver_args[@]}" "${EXPAT_BUILDDIR}"/run.sh "$@"
```

Comment:
Is this a bug fix or clean-up?

Reply:
It is to allow out-of-tree builds.  Strictly speaking: only the variable definition of `top_builddir` needs to be changed with respect to the original script, because the definition of `top_srcdir` looks to work.  But why run an the external program `dirname` and use the expensive `type -p` when a simple environment variable will do? (And Automake has built-in support for setting the environment or the `check` rule.)